# 请求部分（空）
---
# 响应部分
std_msgs/String label
geometry_msgs/Point position

